src/lib/dns-server/service_widget_helpers.rb

```diff
@@ -0,0 +1,22 @@
+require "cwm/service_widget"
+require "yast2/system_service"
+
+module Y2DnsServer
+  module ServiceWidgetHelpers
+    # Returns the 'named' system service
+    #
+    # @return [Yast2::SystemService] 'named' system service
+    def service
+      @service ||= Yast2::SystemService.find("named")
+    end
+
+    # Returns the status widget for service
```

Comment:
Only for curiosity, why do you refer the widget as "status widget"? It confuses me a bit.

Reply:
Because it is the (service) widget responsible to manage the final status of service (running, stopped, etc). But if it is confusing I could change it and simply use "service widget" :)